lib/plugins/aws/invokeLocal/index.js

```diff
@@ -52,10 +52,31 @@ class AwsInvokeLocal {
   }
 
   loadEnvVars() {
+    const lambdaName = this.options.functionObj.name;
+    const MemorySize = Number(this.options.functionObj.memorySize)
+      || Number(this.serverless.service.provider.memorySize)
+      || 1024;
+
+    const lambdaDefaultEnvVars = {
+      PATH: '/usr/local/lib64/node-v4.3.x/bin:/usr/local/bin:/usr/bin/:/bin',
```

Comment:
@pmuens this has nothing to do with the user's machine...those are related to the current Lambda runtime, which is Node + Linux.

Generally though, this is just a mock, and doesn't really represent anything locally, and I'd actually argue that this env var wouldn't be at all useful locally, but at least it won't be undefined.